examples/medplum-scheduling-demo/src/bots/example/example-data.ts

```diff
@@ -0,0 +1,349 @@
+import { BotEvent, createReference, getReferenceString, MedplumClient } from '@medplum/core';
+import {
+  Appointment,
+  Bundle,
+  BundleEntry,
+  CodeableConcept,
+  Encounter,
+  Patient,
+  Practitioner,
+  Reference,
+  Schedule,
+  Slot,
+} from '@medplum/fhirtypes';
+import { randomUUID } from 'crypto';
+
+export async function handler(medplum: MedplumClient, event: BotEvent<Practitioner>): Promise<Bundle> {
+  const practitioner = event.input as Practitioner;
+
+  // Note that the Schedule resource is created in the App.tsx file
+  const schedule = await medplum.searchOne('Schedule', { actor: getReferenceString(practitioner) });
+  if (!schedule) {
+    throw new Error('Schedule not found');
+  }
+
+  const entries: BundleEntry<Patient | Slot | Appointment | Encounter>[] = [...createPatientEntries()];
+
+  const practitionerReference = createReference(practitioner);
+  const homerReference = { reference: 'urn:uuid:bd90afcc-4f44-4c13-8710-84ddc1bce347', display: 'Homer Simpson' };
+  const margeReference = { reference: 'urn:uuid:eca66352-415c-4dab-add1-e4ed8a156408', display: 'Marge Simpson' };
+  const scheduleReference = createReference(schedule);
+
+  const today = new Date();
+
+  // Create slots and appointments for the previous week, this week, and next week
+  createWeekSlots(today, -1, entries, scheduleReference, practitionerReference, [margeReference, homerReference]);
+  createWeekSlots(today, 0, entries, scheduleReference, practitionerReference, [margeReference, homerReference]);
+  createWeekSlots(today, 1, entries, scheduleReference, practitionerReference, [margeReference, homerReference]);
+
+  const bundle: Bundle = {
+    resourceType: 'Bundle',
+    type: 'batch',
+    entry: entries,
+  };
+  return bundle;
```

Comment:
I'm returning the bundle in the bot so that the `uploadExampleData` function aligns with the approach used in other demo apps. Does this seem reasonable, or would it be better to run `medplum.executeBatch` here instead?